streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -391,6 +391,8 @@
     static {
         final Map<String, Object> tempProducerDefaultOverrides = new HashMap<>();
         tempProducerDefaultOverrides.put(ProducerConfig.LINGER_MS_CONFIG, "100");
+        tempProducerDefaultOverrides.put(ProducerConfig.RETRIES_CONFIG, Integer.MAX_VALUE);
+        tempProducerDefaultOverrides.put(ProducerConfig.ACKS_CONFIG, "all");
```

Comment:
I think a KIP unnecessarily slows things down. Why do we need to do a KIP to correct an internal flaw?